src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -272,10 +272,7 @@ IN_PROCESS_APPLICATION::LoadManagedApplication
                 goto Finished;
             }
 
-            if (FAILED(hr = m_pLoggerProvider->Start()))
-            {
-                goto Finished;
-            }
+            LOG_IF_FAILED(hr = m_pLoggerProvider->Start());
```

Comment:
Is this https://github.com/aspnet/IISIntegration/issues/976? Is there a test?

Reply:
Thinking of how to write a test. Will be tricky.

Reply:
Just put a random non existent path into stdout location

Reply:
We create the folder path if it doesn't exist. My current thought is to force invalid permissions. This may be tricky for full IIS as we run as local system/admin.

Reply:
just use disk name that doesn't exist or use invalid path characters.